### PR TITLE
File sync for envd-server

### PR DESCRIPTION
**Pull request by AlexXi19**
Draft pull request for https://github.com/tensorchord/envd-server/issues/102

### CODE REVIEW

File: pkg/syncthing/client.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+const (
+	GET  = "GET"
+	POST = "POST"
+	PUT  = "PUT"
+)
+
+// Makes API calls to the syncthing instance
+func (s *Syncthing) ApiCall(method string, url string, params map[string]string, body []byte) ([]byte, error) {
```
Comment (VoVAllen) [inline pkg/syncthing/client.go:25]: Is this code borrowed from other repo? Please keep license head if so.

Comment (VoVAllen) [reply pkg/syncthing/client.go:25]: Also the client style is not consistent with our current HTTP client

**Review by VoVAllen — COMMENTED** [review]
The client part seems need more discussions

File: pkg/syncthing/client.go (hunk 1 above)
Comment (AlexXi19) [reply pkg/syncthing/client.go:25]: Ok, yeah this is borrowed for now, I was going to ask you about the http client after you wake up.

**Pull request closed without merging**